authentication/auth.go

```diff
@@ -642,9 +641,10 @@ func GetSurveyData(launcherSchema surveys.LauncherSchema) (QuestionnaireSchema,
 
 	missingClaims := getMissingMandatoryClaims(claims, mandatoryClaims)
 
-	for _, v := range missingClaims {
-		schema.Metadata = append(schema.Metadata, v)
-	}
+	//for _, v := range missingClaims {
```

Comment:
How come this is commented out? Do we need this?

Reply:
No we don't need it anymore, I commented it out for testing but forgot to delete it 😆 will remove it 👍